### PR TITLE
Windows server 2012 Install graph

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
test this please

**Comment by benbp** [comment]
👍

**Comment by anhou** [comment]
@tannoa2 I saw you have a documentation stroy https://www.pivotaltracker.com/story/show/119184947 for this workflow. Just a reminder that, except  http://rackhd.readthedocs.io/en/latest/rackhd/index.html#os-installation-workflow-support, it also has a batch of samples at https://github.com/RackHD/RackHD/tree/master/example/samples  for users to refer to. it includes one minimal payload example, and one full set of parameters payload.

**Comment by anhou** [comment]
@tannoa2  we have a naming convention like below, don't have a OS version suffix, and choose to list in docs's version column.  http://rackhd.readthedocs.io/en/latest/rackhd/index.html#os-installation-workflow-support. I'm not sure this workflow's templates or somethings are much different with other windows server version? if not, Is it possible for them to leverage one workflow?
```
OS	        Workflow                 	Version
ESXi	Graph.InstallEsx	        5.5/6.0
RHEL	Graph.InstallRHEL	7.0
CentOS	Graph.InstallCentOS	6.5/7
Ubuntu	Graph.InstallUbuntu	trusty(14.04)
SUSE	Graph.InstallSUSE	openSUSE: leap/42.1, SLES: 11/12
CoreOS	Graph.InstallCoreOS	899.17.0
```

**Comment by tannoa2** [comment]
@anhou  I beleive that we can use the same workflow to install different windows images. 
But we would know for sure until we try with different flavors of windows.
I will add  windows to the supported OS installation list when I work on the documentation story.

**Comment by tannoa2** [comment]
@anhou I will add the a minimum and full payload to the examples

**Comment by anhou** [comment]
@tannoa2 Thanks for confirmation, so will you change graph name to 'Graph.InstallWindowsServer' ,

**Comment by tannoa2** [comment]
@anhou correct

**Comment by anhou** [comment]
:+1:

File: lib/graphs/install-windows-server2012-graph.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Install Windows Server 2012',
+    injectableName: 'Graph.Install',
```
Comment (anhou) [inline lib/graphs/install-windows-server2012-graph.js]: a mistake missing 'WindowsServer'?

**Pull request merged**